Add example and help for gum log --time option to README (#472)
charmbracelet/gum: README.md

# Log some error.
gum log --structured --level error "Unable to create file." name file.txt
# ERROR Unable to create file. name=temp.txt

# Include a timestamp.
gum log --time rfc822 --level error "Unable to create file."
```

> See the Go [`time` package] for acceptable `--time` formats.

Reviewer: ```suggestion
See the Go [`time` package](https://pkg.go.dev/time#pkg-constants) for acceptable `--time` formats.
```

> The `--time` option accepts any constant defined in [the Go `time` package](https://pkg.go.dev/time#pkg-constants), case-insensitively. 

Reviewer: ```suggestion
See the Go [`time` package] for acceptable `--time` formats.
```
